### PR TITLE
CRW-1094 update to...

### PR DESCRIPTION
**Pull request by nickboldt**
CRW-1094 update to redhat/dependency-analytics/0.1.0
CRW-1094 update to redhat/java/0.63.0 (and java8 and java11)
CRW-1094 update to redhat/quarkus-java11/1.5.0
CRW-1094 update to redhat/vscode-apache-camel/0.0.25
CRW-1094 update to redhat/vscode-camelk/0.0.15
CRW-1094 update to sonarsource/sonarlint-vscode/1.16.0

Change-Id: I941684c54dfd51cfe3fad39b9b19c5fd345cbf28
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

File: dependencies/che-plugin-registry/v3/plugins/redhat/java11/latest.txt [hunk 1]
```diff
@@ -1 +1 @@
-0.62.0
```
Comment (ericwill) [inline dependencies/che-plugin-registry/v3/plugins/redhat/java11/latest.txt:1]: We had 0.62.0 downstream but not upstream?

Comment (nickboldt) [reply dependencies/che-plugin-registry/v3/plugins/redhat/java11/latest.txt:1]: yeah, we're awesome.

Comment (ericwill) [reply dependencies/che-plugin-registry/v3/plugins/redhat/java11/latest.txt:1]: Okay. Just be cautious updating ahead of upstream, for example now with `vscode-java` they are switching to Java 11 as a min version, and we haven't adapted to it upstream yet. So going ahead in CRW may result in some challenges there. :)

**Comment by ericwill** [comment]
> > I'm assuming the removals (like PHP) are for plugins deprecated during some previous release?
> 
> I just removed stuff from the 2.0 release era since that's no longer supported. Or that was the intent. Did you see anything super fresh?

Nope just curious.

**Pull request merged**